tgnms/.eslintrc.yml

```diff
@@ -0,0 +1,84 @@
+---
+root: true
+
+parser: fb-eslint
+extends:
+  - eslint-config-fbcnms
+
+plugins:
+  - eslint-config-fbcnms
+  - eslint-config-fb-strict
+  - eslint-plugin-header
+  - eslint-plugin-import
+  - eslint-plugin-node
+  - eslint-plugin-lint
+  - eslint-plugin-sort-imports-es6-autofix
+
+overrides:
+  - files:
+    - '*'
+    rules:
+      prettier/prettier:
+        - 2
+        - singleQuote: true
+          trailingComma: all
+          bracketSpacing: false
+          jsxBracketSameLine: true
+          parser: flow
+  - files:
+      - .eslintrc.js
+    rules:
+      quotes:
+        - warn
+        - single
+  - env:
+      jest: true
+      node: true
+    files:
+      - '**/__mocks__/**/*.js'
+      - '**/__tests__/**/*.js'
+      - '**/tests/**/*.js'
+      - testHelpers.js
+      - testData.js
+      - jest-setup.js
+  - files:
+      - fbcnms-packages/**/*.js
+    rules:
+      header/header:
+        - 2
+        - block
+  - env:
+      node: true
+    files:
+      - .eslintrc.js
+      - babel.config.js
+      - jest.config.js
+      - jest.*.config.js
+      - fbcnms-packages/eslint-config-fbcnms/**/*.js
```

Comment:
we can remove every `fbcnms-packages` line that isn't part of the tgnms repo.

Reply:
removed